inst/shiny/DiagnosticsExplorer/R/StartUpScripts.R

```diff
@@ -408,6 +408,20 @@ initializeEnvironment <- function(shinySettings,
       unique() %>%
       sort()
   }
+  
+  if (!is.null(envir$conceptSets)) {
```

Comment:
What is the reason this is included here? The reason this function exists is because there were hundreds of hacks that changed the global namespace. I want to remove it in a future version as this is bad practice for an app this large.